http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtSdkHttpService.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.http.crt;
+
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.http.async.SdkAsyncHttpClient;
+import software.amazon.awssdk.http.async.SdkAsyncHttpService;
+
+/**
+ * Service binding for the AWS common runtime HTTP client implementation. Allows SDK to pick this up automatically from the
+ * classpath.
+ */
+@SdkPublicApi
+public class AwsCrtSdkHttpService implements SdkAsyncHttpService {
```

Comment:
We need a matching services file for this as well

https://github.com/aws/aws-sdk-java-v2/blob/20a9f40e44df19c18017517eb1ceac2e5ec6e86e/http-clients/netty-nio-client/src/main/resources/META-INF/services/software.amazon.awssdk.http.async.SdkAsyncHttpService#L1-L16

Reply:
Good catch. Added